docs/src/modules/components/Demo.js

```diff
@@ -168,6 +170,20 @@ function Demo(props) {
     setDemoHovered(event.type === 'mouseenter');
   };
 
+  const [SnackbarOpen, setSnackbarOpen] = React.useState(false);
+  const [SnackBarMessage, setSnackBarMessage] = React.useState(undefined);
+
+  const Alert = SnackBarProps => {
+    return <MuiAlert elevation={6} variant="filled" {...SnackBarProps} />;
```

Comment:
So long as Material-UI remains "Material Design first", I think we should stick to the non-customized Snackbar.